common/scala/src/main/scala/whisk/core/containerpool/logging/ElasticSearchRestClient.scala

```diff
@@ -58,6 +58,7 @@ case class EsQueryRange(key: String, range: EsRange, value: String)
 case class EsQueryBoolMatch(key: String, value: String)
 case class EsQueryOrder(field: String, kind: EsOrder)
 case class EsQuerySize(size: Integer)
+case class EsQueryFrom(from: Integer)
```

Comment:
I was thinking the other day: Can both "size" and "from" just be `Int` (without the wrapping type)?

On the same note: Make those `Int` to avoid boxing.

Reply:
Seems like that worked. 👍

Reply:
What about removing the wrapping type for both?

Reply:
Which one am I missing?